src/main/java/com/atlassian/bitbucket/jenkins/internal/status/BitbucketSCMAction.java

```diff
@@ -0,0 +1,41 @@
+package com.atlassian.bitbucket.jenkins.internal.status;
+
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCM;
+import hudson.model.Action;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * Holds the Bitbucket SCM that was used to checkout the sources. This is needed when build status is posted to find
+ * the project key and repository slug.
+ */
+public class BitbucketSCMAction implements Action {
+
+    private final BitbucketSCM bitbucketSCM;
+
+    public BitbucketSCMAction(BitbucketSCM scm) {
+        bitbucketSCM = scm;
+    }
+
+    public BitbucketSCM getBitbucketSCM() {
+        return bitbucketSCM;
+    }
+
+    @CheckForNull
+    @Override
+    public String getDisplayName() {
+        return null;
```

Comment:
Should these be null?

Reply:
yes, otherwise the action shows up in the UI, so all the inherited methods needs to be null